[2023-06-27T10:49:37.685+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-27T10:48:00+00:00/task_id=create_select permission to 509
[2023-06-27T10:49:37.686+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-27T10:48:00+00:00/task_id=create_select permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-27T10:48:00+00:00/task_id=create_select'
[2023-06-27T10:49:37.724+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-27T10:48:00+00:00/task_id=create_select permission to 509
[2023-06-27T10:49:37.726+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-27T10:48:00+00:00/task_id=create_select permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-27T10:48:00+00:00/task_id=create_select'
[2023-06-27T10:49:37.899+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Loop.create_select scheduled__2023-06-27T10:48:00+00:00 [queued]>
[2023-06-27T10:49:37.911+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Loop.create_select scheduled__2023-06-27T10:48:00+00:00 [queued]>
[2023-06-27T10:49:37.912+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-27T10:49:37.937+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): create_select> on 2023-06-27 10:48:00+00:00
[2023-06-27T10:49:37.945+0000] {standard_task_runner.py:57} INFO - Started process 72 to run task
[2023-06-27T10:49:37.949+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '2_Test_Badge_5_Loop', 'create_select', 'scheduled__2023-06-27T10:48:00+00:00', '--job-id', '242', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpafietp73']
[2023-06-27T10:49:37.952+0000] {standard_task_runner.py:85} INFO - Job 242: Subtask create_select
[2023-06-27T10:49:38.008+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-27T10:48:00+00:00/task_id=create_select permission to 509
[2023-06-27T10:49:38.009+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-27T10:48:00+00:00/task_id=create_select permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-27T10:48:00+00:00/task_id=create_select'
[2023-06-27T10:49:38.012+0000] {task_command.py:410} INFO - Running <TaskInstance: 2_Test_Badge_5_Loop.create_select scheduled__2023-06-27T10:48:00+00:00 [running]> on host 87de987b9766
[2023-06-27T10:49:38.153+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='2_Test_Badge_5_Loop' AIRFLOW_CTX_TASK_ID='create_select' AIRFLOW_CTX_EXECUTION_DATE='2023-06-27T10:48:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-27T10:48:00+00:00'
[2023-06-27T10:49:38.155+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-27T10:49:38.157+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /dbt && dbt run --model toto_select']
[2023-06-27T10:49:38.168+0000] {subprocess.py:86} INFO - Output:
[2023-06-27T10:49:41.565+0000] {subprocess.py:93} INFO - [0m10:49:41  Running with dbt=1.5.1
[2023-06-27T10:49:50.483+0000] {subprocess.py:93} INFO - [0m10:49:50  Found 2 models, 0 tests, 0 snapshots, 0 analyses, 455 macros, 0 operations, 1 seed file, 0 sources, 0 exposures, 0 metrics, 0 groups
[2023-06-27T10:49:50.486+0000] {subprocess.py:93} INFO - [0m10:49:50
[2023-06-27T10:49:52.560+0000] {subprocess.py:93} INFO - [0m10:49:52  Concurrency: 200 threads (target='dev')
[2023-06-27T10:49:52.561+0000] {subprocess.py:93} INFO - [0m10:49:52
[2023-06-27T10:49:52.634+0000] {subprocess.py:93} INFO - [0m10:49:52  1 of 1 START sql table model ***.toto_select ............................... [RUN]
[2023-06-27T10:49:54.216+0000] {subprocess.py:93} INFO - [0m10:49:54  1 of 1 OK created sql table model ***.toto_select .......................... [[32mSUCCESS 1[0m in 1.58s]
[2023-06-27T10:49:54.270+0000] {subprocess.py:93} INFO - [0m10:49:54
[2023-06-27T10:49:54.272+0000] {subprocess.py:93} INFO - [0m10:49:54  Finished running 1 table model in 0 hours 0 minutes and 3.78 seconds (3.78s).
[2023-06-27T10:49:54.304+0000] {subprocess.py:93} INFO - [0m10:49:54
[2023-06-27T10:49:54.305+0000] {subprocess.py:93} INFO - [0m10:49:54  [32mCompleted successfully[0m
[2023-06-27T10:49:54.307+0000] {subprocess.py:93} INFO - [0m10:49:54
[2023-06-27T10:49:54.308+0000] {subprocess.py:93} INFO - [0m10:49:54  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2023-06-27T10:49:55.089+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-27T10:49:55.115+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=2_Test_Badge_5_Loop, task_id=create_select, execution_date=20230627T104800, start_date=20230627T104937, end_date=20230627T104955
[2023-06-27T10:49:55.144+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-27T10:49:55.164+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
